### PR TITLE
[sc-5821] Reset search pagination

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

**Review by ebrehault — APPROVED** [review]

File: libs/search-widget/src/core/search-bar.ts [hunk 1]
```diff
@@ -37,6 +38,7 @@ export const setupTriggerSearch = (
             switchMap(() => searchQuery.pipe(take(1))),
             tap((query) => (dispatch ? dispatch('search', query) : undefined)),
             tap((query) => (!trigger?.more ? ask.next({ question: query, reset: true }) : undefined)),
+            tap(() => (!trigger?.more ? pageNumber.set(0) : undefined)),
```
Comment (mpellerin42) [inline libs/search-widget/src/core/search-bar.ts:41]: In your code, it seems pageNumber can be only `0` or `undefined`, but what when we scroll down to get next page?

Comment (operramon) [reply libs/search-widget/src/core/search-bar.ts:41]: the code maybe is confusing, when scrolling down it doesn't set the page number as trigger.more is `true`

Comment (mpellerin42) [reply libs/search-widget/src/core/search-bar.ts:41]: yes, I got that, but this is actually my point: pageNumber is updated only here on the code, so currently pageNumber's value can be either `0`, either `undefined`. I'm wondering if it should be updated when we load the next pages?

Comment (operramon) [reply libs/search-widget/src/core/search-bar.ts:41]: oh, seems the page number increment is here:
https://github.com/nuclia/frontend/blob/oriolperramon8396/sc-5821/reset-search-pagination/libs/search-widget/src/core/stores/search.store.ts#L137

Comment (mpellerin42) [reply libs/search-widget/src/core/search-bar.ts:41]: oh! this is the part I missed! perfect then :)

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**